addons/editor-theme3/addon.json

```diff
@@ -367,32 +436,10 @@
         "text": "white"
       }
     },
-    {
-      "name": "Dark",
-      "id": "dark",
-      "description": "Dark versions of the default colors",
-      "values": {
-        "motion-color": "#004099",
-        "looks-color": "#220066",
-        "sounds-color": "#752475",
-        "events-color": "#997300",
-        "control-color": "#664100",
-        "sensing-color": "#1f5f7a",
-        "operators-color": "#235c23",
-        "data-color": "#b35900",
-        "data-lists-color": "#993300",
-        "custom-color": "#99004d",
-        "Pen-color": "#064734",
-        "sa-color": "#166966",
-        "comment-color": "#423f24",
-        "input-color": "#202020",
-        "text": "white"
-      }
-    },
     {
       "name": "Black",
       "id": "black",
-      "description": "Makes block backgrounds black",
+      "description": "Makes block backgrounds black and borders/text colored",
```

Comment:
```suggestion
      "description": "Makes block backgrounds black and the borders and text colored",
```

Reply:
Indifferent about this. We aren't that strict with preset descriptions. For example, we don't end the sentences with a period within preset descriptions